### PR TITLE
JC: Update to latest version of program-boilerplate

### PR DESCRIPTION
**Pull request by jayden-chan**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -57,3 +57,4 @@ typings/
 # dotenv environment variables file
 .env
 
+packages/program-boilerplate/src/utils.js
```
[the remark this answers is not in the document]
Comment (jiaqihuang23) [reply .gitignore]: It should be removed from here.

File: package.json [hunk 2]
```diff
@@ -17,6 +17,9 @@
   },
   "homepage": "https://github.com/saasquatch/program-tools#readme",
   "devDependencies": {
-    "lerna": "^2.5.1"
+    "lerna": "^2.11.0"
+  },
+  "dependencies": {
+    "yarn": "^1.7.0"
```
[the remark this answers is not in the document]
Comment (jiaqihuang23) [reply package.json]: Design inherited from earliest version. @loganvolkers

[the remark this answers is not in the document]
Comment (jericmason) [reply package.json]: Going to leave it as it is for now.

File: packages/vanilla-components/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@saasquatch/vanilla-components",
-  "version": "0.0.21",
+  "name": "lerna-test-vanilla",
```
[the remark this answers is not in the document]
Comment (jericmason) [reply packages/vanilla-components/package.json]:
```suggestion
  "name": "@saasquatch/vanilla-components",
```
This should be deleted/reverted, it's not consistent with the current deployed vanilla components and should not affect the automated tests.

File: packages/program-boilerplate/src/conversion.js [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+function parseValue(value) {
+    if (/^(\-|\+)?([0-9]+(\.[0-9]+)?)$/
+      .test(value)) {
+      return Number(value);
+    }
+  
+    if (value.toLowerCase() === "true" || "yes") {
+      return Boolean(true);
+    }
+    if (value.toLowerCase() === "false" || "no") {
```
Comment (jayden-chan) [inline packages/program-boilerplate/src/conversion.js]: Is this expression intended to be written like this? These `if` statements are always going to evaluate to true.

image

Comment (jiaqihuang23) [reply packages/program-boilerplate/src/conversion.js]: No. It's incorrect.

**Pull request merged**